sdk/storage/azure-storage-files-datalake/inc/azure/storage/files/datalake/datalake_options.hpp

```diff
@@ -208,6 +209,50 @@ namespace Azure { namespace Storage { namespace Files { namespace DataLake {
     Azure::Core::Nullable<std::string> Directory;
   };
 
+  /**
+   * @brief Optional parameters for FileSystemClient::GetAccessPolicy.
+   */
+  struct GetDataLakeFileSystemAccessPolicyOptions
+  {
+    /**
+     * @brief Context for cancelling long running operations.
+     */
+    Azure::Core::Context Context;
+
+    /**
+     * @brief Optional conditions that must be met to perform this operation.
+     */
+    LeaseAccessConditions AccessConditions;
+  };
+
+  /**
+   * @brief Optional parameters for FileSystemClient::SetAccessPolicy.
+   */
+  struct SetDataLakeFileSystemAccessPolicyOptions
+  {
+    /**
+     * @brief Context for cancelling long running operations.
+     */
+    Azure::Core::Context Context;
+
+    /**
+     * @brief Specifies whether data in the file system may be accessed publicly and the level
+     * of access.
+     */
+    Azure::Core::Nullable<Models::PublicAccessType> AccessType;
```

Comment:
This is exactly what we've discussed, right? not nullable?

Reply:
Sorry, missed this one.